wiki/Rules/id.md

```diff
@@ -18,7 +18,7 @@ Peraturan-peraturan berikut ini merupakan pedoman dasar yang kami terapkan di li
 3. **Jika di tengah pembicaraan terdapat seseorang yang terputus dari jaringan *(timed out)*, di-silence, atau di-restrict, abaikan saja dan jangan mulai membicarakannya.** Jika seseorang di-restrict, kasus mereka akan otomatis jatuh pada ranah yang hanya melibatkan staf administratif osu! dan diri mereka sendiri.
```

Comment:
Tambahan mungkin ini, orang yg kena restrict mereka gak bisa dilihat oleh siapapun kecuali staff?

Reply:
it's not written in the original Rules document so i don't think it would be a good idea to add this